wsl-builder/common/releasesinfo.go

```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 )
 
-type wslReleaseInfo struct {
+type WslReleaseInfo struct {
```

Comment:
I found the reason why you didn’t see any complain about exported fields: `golint` repository [has been archived](https://github.com/golang/lint) and VSCode extensions seem to (I didn’t find it in the changelog) have disabled it recently.

I think we will use golang-ci linters with a set of default rules similar to the one we have in adsys (switching from `golint` to `revive` thus), but the story won’t be good as gopls would complain it’s not run in project root.

Anyway, documentation should be in format of `ExportedSymbol <verb> …`. As we have been able to supports this in our projects, let’s keep it that way. The same rule goes for any exported field in the struct below (if you need more explanation on some of them, do not hesitate to ask)